### PR TITLE
feat: Advanced Screen (Within Settings Screen): Dynamic Type Support - ACC-99

### PR DESCRIPTION
**Pull request by BrzozowskiPawel**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-99" title="ACC-99" target="_blank">TaskACC-99</a>  [iOS] Advanced Screen (Within Settings Screen): Dynamic Type Support (Partially Implemented)
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR, fixed truncated text in Reset Push Notifications Token label when Dynamic type has been applied.</br>

| iPhone 8 - English  | iPhone 8 - German  |  iPhone SE - English | iPhone SE - German  |
|---|---|---|---|
| Screenshot 2022-05-12 at 08 21 49  | Screenshot 2022-05-12 at 08 26 22  | Screenshot 2022-05-12 at 09 54 36 | Screenshot 2022-05-12 at 08 25 21  |

### Issues

Basically my task was to:
[] Fix the colour of the arrows in cells. As we have been discussing we can wait till min. versions of the app will be 13 and use SF symbol here to do not search for a walkaround of the problem that SF Symbol will easily fix.
[x] Fix truncated text in Reset Push Notifications Token label when Dynamic type has been applied. It has been finished.

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: WireCommonComponents/FontScheme.swift [hunk 1]
```diff
@@ -226,3 +234,113 @@ public enum FontScheme {
...
+    switch (fontSize, contentSizeCategory) {
+    // SMALLL
+    case (.small, .extraSmall):
+        return 14
+    case (.small, .small):
+        return 15
+    case (.small, .medium):
+        return 16
+    case (.small, .large):
+        return 17
```
Comment (johnxnguyen) [inline WireCommonComponents/FontScheme.swift:247]: `UIContentSizeCategory.large` is the default size, and previously the default point size for `FontSize.small` was 11. On the one hand, the design team is rethinking what point sizes to use, but on the other hand, we don't want to change the existing design at this point.

You may notice that although the CI reports a green check, it does so erroneously as there are a lot of test failures. This is expected when the default point sizes are changed.

My suggestion would be to use the default point sizes as a starting point, then determine the maximum and minimum point sizes around that default.

Comment (BrzozowskiPawel) [reply WireCommonComponents/FontScheme.swift:247]: @johnxnguyen Yes, you are right. This is the case that I want to disguise on today's meeting :D

**Review by agisilaos — CHANGES_REQUESTED** [review]

File: WireCommonComponents/FontScheme.swift [hunk 2]
```diff
@@ -226,3 +234,113 @@ public enum FontScheme {
     }
 }
 
+func pointSize(fontSize: FontSize, contentSizeCategory: UIContentSizeCategory) -> CGFloat {
+    switch (fontSize, contentSizeCategory) {
+    // SMALLL
```
[the remark this answers is not in the document]
Comment (BrzozowskiPawel) [reply WireCommonComponents/FontScheme.swift]: @agisilaos as we have talked about this is now part of another feature:
https://github.com/wireapp/wire-ios/pull/5677

But you are more than right I will change it there

File: Wire-iOS.xcodeproj/xcshareddata/xcschemes/Wire-iOS.xcscheme [hunk 3]
```diff
@@ -132,6 +132,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "de"
```
[the remark this answers is not in the document]
Comment (BrzozowskiPawel) [reply Wire-iOS.xcodeproj/xcshareddata/xcschemes/Wire-iOS.xcscheme:135]: Yes of course. It's not set to system language as it was on default.
I will focus more to do not push things like this anymore.

**Review by johnxnguyen — APPROVED** [review]
Looks good, but don't forget to remove the changes for the language in the scheme.

**Review by agisilaos — APPROVED** [review]
